tests/console/docker_image.pm

```diff
@@ -55,54 +59,59 @@ sub run {
     }
```

Comment:
Do we want to install container-diff also for public cloud case? It will be installed on the openqa instance, not on the PC instance, so maybe you can add condition here too.

Reply:
I would be inclined to do this in a different PR otherwise we include too many things at once

Reply:
In my opinion the whole `docker_image.pm` will not work here because the `registry.suse.de` is not accessible from outside.

Reply:
`docker-image.pm` looks pretty empty on the public cloud runs: http://phoenix-openqa.qam.suse.de/tests/1576

Reply:
Yes. Just exclude it for now.

Reply:
Excluded if `@$image_names` is empty

Reply:
same for podman_image

Reply:
Updated accordingly